### PR TITLE
Release v1.2.0

### PR DESCRIPTION
**Pull request by claytonparnell**
*Issue #, if available:*

*Description of changes:* Adds NodeJS, CodeWhisperer

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by claytonparnell** [comment]
# Staleness Report: 1.2.0(gpu)

Package | Current Version in the Distribution image | Latest Relevant Version in Upstream
---|---|---
numpy|1.26.0|1.26.0
jinja2|3.1.2|3.1.2
pandas|2.1.2|2.1.2
altair|5.1.2|5.1.2
${\color{red}boto3}$|1.28.64|1.28.81
ipython|8.17.2|8.17.2
jupyter-lsp|2.2.0|2.2.0
jupyterlab|4.0.8|4.0.8
amazon-codewhisperer-jupyterlab-ext|2.0.0|2.0.0
jupyter-scheduler|2.3.0|2.3.0
amazon-sagemaker-jupyter-scheduler|3.0.4|3.0.4
scipy|1.11.3|1.11.3
scikit-learn|1.3.2|1.3.2
pip|23.3.1|23.3.1
torchvision|0.15.2|0.15.2
autogluon|0.8.2|0.8.2
ipywidgets|8.1.1|8.1.1
notebook|7.0.6|7.0.6
aws-glue-sessions|1.0.2|1.0.2
conda|23.10.0|23.10.0
jupyter-ai|2.5.0|2.5.0
jupyter-dash|0.4.2|0.4.2
jupyter-server-proxy|4.1.0|4.1.0
${\color{red}keras}$|2.12.0|2.14.0
matplotlib|3.8.1|3.8.1
nodejs|18.17.1|18.17.1
py-xgboost-gpu|1.7.6|1.7.6
python-lsp-server|1.9.0|1.9.0
${\color{red}pytorch-gpu}$|2.0.0|2.1.0
sagemaker-headless-execution-driver|0.0.9|0.0.9
sagemaker-jupyterlab-emr-extension|0.1.9|0.1.9
sagemaker-jupyterlab-extension|0.1.9|0.1.9
sagemaker-kernel-wrapper|0.0.2|0.0.2
sagemaker-python-sdk|2.197.0|2.197.0
sagemaker-studio-analytics-extension|0.0.21|0.0.21
supervisor|4.2.5|4.2.5
${\color{red}tensorflow}$|2.12.1|2.13.1

# Staleness Report: 1.2.0(cpu)

Package | Current Version in the Distribution image | Latest Relevant Version in Upstream
---|---|---
numpy|1.26.0|1.26.0
jinja2|3.1.2|3.1.2
${\color{red}pytorch}$|2.0.0|2.1.0
pandas|2.1.2|2.1.2
altair|5.1.2|5.1.2
${\color{red}boto3}$|1.28.64|1.28.81
ipython|8.17.2|8.17.2
jupyter-lsp|2.2.0|2.2.0
jupyterlab|4.0.8|4.0.8
amazon-codewhisperer-jupyterlab-ext|2.0.0|2.0.0
jupyter-scheduler|2.3.0|2.3.0
amazon-sagemaker-jupyter-scheduler|3.0.4|3.0.4
scipy|1.11.3|1.11.3
scikit-learn|1.3.2|1.3.2
pip|23.3.1|23.3.1
torchvision|0.15.2|0.15.2
autogluon|0.8.2|0.8.2
ipywidgets|8.1.1|8.1.1
notebook|7.0.6|7.0.6
aws-glue-sessions|1.0.2|1.0.2
conda|23.10.0|23.10.0
jupyter-ai|2.5.0|2.5.0
jupyter-dash|0.4.2|0.4.2
jupyter-server-proxy|4.1.0|4.1.0
${\color{red}keras}$|2.12.0|2.14.0
matplotlib|3.8.1|3.8.1
nodejs|18.17.1|18.17.1
py-xgboost-cpu|1.7.6|1.7.6
python-lsp-server|1.9.0|1.9.0
sagemaker-headless-execution-driver|0.0.9|0.0.9
sagemaker-jupyterlab-emr-extension|0.1.9|0.1.9
sagemaker-jupyterlab-extension|0.1.9|0.1.9
sagemaker-kernel-wrapper|0.0.2|0.0.2
sagemaker-python-sdk|2.197.0|2.197.0
sagemaker-studio-analytics-extension|0.0.21|0.0.21
supervisor|4.2.5|4.2.5
${\color{red}tensorflow}$|2.12.1|2.13.1

**Review by navinsoni — APPROVED** [review]

**Review by just4brown — APPROVED** [review]
